spec/reporters/worker_reporter_spec.js

```diff
@@ -0,0 +1,22 @@
+var WorkerReporter = require('../../lib/reporters/worker_reporter');
+
+describe("WorkerReporter", function() {
+  beforeEach(function() {
+    this.reporter = new WorkerReporter();
+    process.send = this.fakeSend = jasmine.createSpy("fakeSend");
```

Comment:
This should probably `spyOn` `process.send` instead of clobbering it manually. This way it will get cleaned up after the spec(s) and not leak into other.

Reply:
See [process](https://nodejs.org/api/process.html#process_process_send_message_sendhandle_options_callback
) If Node.js was not spawned with an IPC channel, process.send() will be undefined.

In my test, I do not create any `child_process`

Reply:
Fair enough. I think we might want to set it back to `undefined` in an `afterEach` then just to ensure the environment is clean again.